src/ms-review.tex

```diff
@@ -0,0 +1,1785 @@
+to WCS, HEALPix and region based data structures. The reduced data is bundled
+into datasets, which serve as a basis for Poisson maximum likelihood
+modelling of the data. For this purpose \gammapy provides a wide selection
+of built-in, spectral, spatial and temporal models as well as unified
+fitting interface with connection to multiple optimization backends.
+
+With the v1.0 milestone the \gammapy project enters a new development
+phase. Future work will not only include maintenance of the v1.0 release,
+but also parallel development of new features, improved API and data
+model support. While v1.0 provides all the features required for
+standard and advanced astronomical \gammaray data analysis,
+we already identified specific improvements to be considered in the
+roadmap for a future v2.0 release. This includes the support for
+scalable analyses via distributed computing. This will allow
+users to scale an analysis from a few observations to multiple
+hundreds of observations as expected by deep surveys of the CTA
+observatory. In addition the high level interface
+of \gammapy is planned to be developed into a fully configurable
+API design. This will allow users to define arbitrary complex analysis
+scenarios as YAML files and even extend their workflows by user defined
+analysis steps via a registry system. Another important topic will
+be to improve the support of handling meta data for data structures
+and provenance information to track the history of the data reduction
+process from the DL3 to the highest DL5/DL6 data levels.
+
+Around the core Python package a large diverse community of
+users and contributors has developed. With regular developer meetings,
+coding sprints and in-person user tutorials at relevant conferences
+and collaboration meetings, the community has constantly grown.
+So far \gammapy has seen ~80 contributors from ~10 different countries.
+With typically ~10 regular contributors at any given time of the
+project, the code base has constantly grown its range of features
+and improved its code quality. With \gammapy being selected as the base library
+for the future science tools for CTA, we expect the community to grow
+even further, providing a stable perspective for further usage,
+development and maintenance of the project. Besides the future use
+by the CTA community \gammapy has already
+been used for analysis of data from the \hess and \magic instruments
+
+While \gammapy was mainly developed for the science community around
+IACT instruments, the data model and software design is general
+enough to be applied to other \gammaray instruments as well.
+The use of \gammapy for the analysis of data from the High Altitude
+Water Cherenkov Observatory (HAWC) has been successfully
+demonstrated by \todo{Olivera et al.}. This makes \gammapy
+a viable choice for the base library for the science tools
+of the future Southern Widefield Gamma Ray Observatory
+(SWGO) and use with data from LHAASO as well. \gammapy
```

Comment:
Proposal of addition:
As very high-energy neutrino observatories share the same analysis flow as the non-pointing instruments like HAWC, SWGO and LHAASO, the Cubic Kilometre Neutrino Telescope (KM3NeT) community is also experiencing \gammapy, which would extend it to a multi-messenger analysis tool for the very high-energy astrophysics.

Reply:
Done